source/features/useful-not-found-page.tsx

```diff
@@ -170,26 +166,46 @@ async function showAlternateLink(): Promise<void> {
 	url.assign({route: 'blob', branch: fileChanges.commit.parentSha, filePath: url.filePath});
 	const lastVersionUrl = fileChanges.file.status === 'removed' ? fileChanges.file.blob_url : url.href;
 	const lastVersion = <a href={lastVersionUrl}>This {getType()}</a>;
-	const permalink = <a href={fileChanges.commit.url}><relative-time datetime={fileChanges.commit.date}/></a>;
+	const permalink = (
+		<a href={fileChanges.commit.url}>
+			<relative-time datetime={fileChanges.commit.date}/>
+		</a>
+	);
 	const verb = fileChanges.file.status === 'removed'
 		? 'deleted'
 		: <a href={fileChanges.file.blob_url}>moved</a>;
 
-	select('main > .container-lg')!.before(
+	return (
 		<p className="container mt-4 text-center">
 			{lastVersion} was {verb} ({permalink}) - {commitHistory}.
-		</p>,
+		</p>
 	);
 }
 
+async function showGitObjectHistory(): Promise<void> {
+	const link = await getGitObjectHistoryLink();
+	if (link) {
+		select('main > .container-lg')!.before(link);
+	}
+}
+
+async function showGitObjectHistoryOnRepo(description: HTMLDivElement): Promise<void> {
+	const link = await getGitObjectHistoryLink();
+	if (link) {
+		link.className = description.className;
+		link.style.display = 'inline';
```

Comment:
Their DOM sucks:

<img width="356" alt="Screenshot 9" src="https://github.com/refined-github/refined-github/assets/1402241/46cba029-c43e-4262-a709-c2bd8cd74497">

They're hiring just about anyone at GitHub. What a dumpster fire. Let's just use utility classes:


```suggestion
		link.className = 'color-fg-muted';
```

Reply:
🤔 How can we resolve this CLS issue? using skeleton?
![화면 기록 2023-08-27 23 47 12](https://github.com/refined-github/refined-github/assets/50487467/b59ca185-3fcb-433b-b06b-101b890ca258)